domain/model/state/controllerstate_test.go

```diff
@@ -1872,3 +1872,11 @@ func (s *stateSuite) TestGetControllerModelUUIDNotFound(c *gc.C) {
 	_, err := modelSt.GetControllerModelUUID(context.Background())
 	c.Check(err, jc.ErrorIs, modelerrors.NotFound)
 }
+
+func (s *stateSuite) TestGetActivatedModelUUIDs(c *gc.C) {
```

Comment:
It would be good to have a test where there is no activated model either in the controller or in the set passed in. We should test our worst cases as well.

Reply:
Sure will add those, thanks